andela/phlox-ah-frontend
# #160451464 user should be able to view tags

What does this PR do?
This PR adds the page to view tags and the articles under a tag

Description of Task to be completed?
Show tags on the sidebar
#### How should this be manually tested?
Go to the root url, you should see a sidebar showing tags.
On clicking a tag, you should see articles under that tag
#### Any background context you want to provide?
None
### Screenshots

#
<img width="1267" alt="screen shot 2018-11-21 at 10 41 03 am" src="https://user-images.githubusercontent.com/23643904/48832693-0924cd80-ed7a-11e8-82ea-ad7c2cbff56d.png">
### What are the relevant pivotal tracker stories?
#160451464
